composer.json

```diff
@@ -0,0 +1,38 @@
+{
+    "name": "silverstripe/tx-translator",
+    "description": "Silverstripe TX Translator",
+    "type": "silverstripe-vendormodule",
+    "authors": [
+        {
+            "name": "SilverStripe",
+            "homepage": "http://silverstripe.com"
+        },
+        {
+            "name": "The SilverStripe Community",
+            "homepage": "http://silverstripe.org"
+        }
+    ],
+    "require": {
+        "php": "^7.4 || ^8.1",
+        "silverstripe/framework": "^4.12 || ^5.0",
+        "silverstripe/vendor-plugin": "^1 || ^2",
```

Comment:
```suggestion
```
It doesn't actually use either of these. If you want to make sure it's _not_ installable with a cms <= 4.11 project, you can add that as a `conflict`

Reply:
Updated